lib/ansible/module_utils/vmware.py

```diff
@@ -275,7 +433,8 @@ def list_snapshots_recursively(snapshots):
     snapshot_data = []
     for snapshot in snapshots:
         snapshot_data.append(deserialize_snapshot_obj(snapshot))
-        snapshot_data = snapshot_data + list_snapshots_recursively(snapshot.childSnapshotList)
+        snapshot_data = snapshot_data + \
+            list_snapshots_recursively(snapshot.childSnapshotList)
```

Comment:
This should not be necessary for PEP8. We have a line-length of 160, not the default 80.